src/tree/localProject/LocalFunctionsTreeItem.ts

```diff
@@ -34,7 +34,9 @@ export class LocalFunctionsTreeItem extends FunctionsTreeItemBase {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        if (this.parent.isIsolated) {
+        const isParentPythonV2Plus = this.parent.langauge === ProjectLanguage.Python && this.parent.languageModel && this.parent.languageModel > 1;
```

Comment:
Can we reuse the `isPythonV2Plus` utility?

Reply:
Consolidated.